data/functions/itemized.sql

```diff
@@ -16,3 +124,22 @@ begin
     perform retry_processing_schedule_b_records(:START_YEAR_AGGREGATE);
 end
 $$ language plpgsql;
+
+CREATE OR REPLACE FUNCTION rename_table_cascade(table_name TEXT) RETURNS VOID AS $$
+DECLARE
+    child_tmp_table_name TEXT;
+    child_tables_cursor CURSOR (parent TEXT) FOR
+        SELECT c.oid::pg_catalog.regclass AS name
+        FROM pg_catalog.pg_class c, pg_catalog.pg_inherits i
+        WHERE c.oid=i.inhrelid
+            AND i.inhparent = (SELECT oid from pg_catalog.pg_class rc where relname = parent);
```

Comment:
@vrajmohan I have a sense of what's going on here but am not as familiar with Postgres' system tables/objects.  The general setup here to grab all child tables that share the same base name as the given parent table and setup the result of that as a cursor to then be looped through below, right?